app/views/ui-components/v1/filters/_plan_filters.html.erb

```diff
@@ -257,13 +257,20 @@
     <% end %>
     <div class="d-flex mb-md-4 col-sm px-0">
       <div class="mr-auto col-sm col-md-6 col-lg-6 p-0 filter-input-block">
-        <label for="hsa_eligibility"><%= render partial:"shared/glossary", locals: {key: "hc4cc_eligible_title_info", term: l10n("hsa_eligible"), question_mark: true } %></label>
+      <fieldset class="filter-input-block d-block">
+        <legend ><%= render partial:"shared/glossary", locals: {key: "hc4cc_eligible_title_info", term: l10n("hsa_eligible"), question_mark: true } %></legend>
```

Comment:
small question - legends have a heavier/bold font weight applied from our styles. do we want this? if not, apply class "weight-n"

Reply:
Looks like the mockup has the font weight at 600 which is weight for the legend. This markup was the best way to handle tooltips on the form value. Typically it would be a label but that wasn't working correctly with the tooltips.

https://www.figma.com/design/Z1WtgrjH3zB9Y7HQVn0hL5/EA-nuharbor-1?node-id=579-59694&t=pbiV3W2wcd8fgPob-1